plugins/ports-plugin/README.md

```diff
@@ -0,0 +1,15 @@
+# Ports Plug-in
+This plug-in is checking network and detect when new ports are listening or closed.
+
+If a process is listening, it will check
+- if listening interface is 127.0.0.1/::1 it asks the user to create a redirect for this port so it will externally available
+- if listening remotely:
+     - if there is a workspace server that is matching for this port, display the URL of this server and propose to open the link.
```

Comment:
Wouldn't it be better to talk about `pre-exposed endpoint` rather than `workspace server`? And `display the URL of this endpoint`?

Reply:
@l0rd well in that case it's not really a `pre-exposed endpoint`
For example a user has specified in its workspace configuration a server, let say 'express' with port `3000` in that case we just show the public URL when the port is listening.
But of course I can change the wording. But this bullet was referencing the case of a valid workspace that is exposing ports as expected. In that case the plug-in does only 'show public URL'

Reply:
So it's not `pre-exposed` I had misunderstood. But anyway it's the `workspace server` that is a Che 6 terminology. In the devfile and che-plugin.yaml we will call it endpoint I think so we may use the new term here as well.